cfgov/v1/jinja2/v1/includes/article.html

```diff
@@ -38,8 +37,7 @@
 
 <article>
     <header>
-        {% set data = {
-            'category': page.categories.all(),
```

Comment:
Note explicit removal of this `'category'` parameter here. This wasn't doing anything because the item-introduction.html template wasn't using a `value.category` parameter, but was getting categories from the page object.

This bug prevents categories from showing up on blog or newsroom posts, and has been broken since #2450 in 2016. See internal https://github.local/CFGOV/platform/issues/3790 that was opened in 2020 about this and closed in 2022.

Having just revisited that code for this PR, I want to flag this for @sonnakim @csebianlander in case there is interest in re-adding categories to those page types.

Reply:
What @csebianlander said - makes sense to me, too